antora-ui-camel/src/css/vars.css

```diff
@@ -150,6 +150,8 @@
   --doc-max-width--desktop: calc(1366 / var(--rem-base) * 1rem);
   --static-max-width: calc(720 / var(--rem-base) * 1rem);
   --static-max-width--desktop: calc(1366 / var(--rem-base) * 1rem);
+  --scrollbar-thickness: calc(7.56 / var(--rem-base) * 1rem);
+  --scrollbar-radius: calc(56.25 / var(--rem-base) * 1rem);
```

Comment:
I'd move this with the other `--scrollbar` variables. This is somewhat confusing calculation, perhaps use the `em` like in `--body-font-size*` and add a comment as to how many pixels that corresponds to.